go.mod

```diff
@@ -46,9 +47,9 @@ require (
 	github.com/mattn/go-colorable v0.1.0
 	github.com/mcuadros/go-version v0.0.0-20180611085657-6d5863ca60fa
 	github.com/pkg/errors v0.8.1
-	github.com/rancher/kontainer-driver-metadata v0.0.0-20191125162457-09a06565759f
+	github.com/rancher/kontainer-driver-metadata v0.0.0-20191210185103-dc45b432da60
```

Comment:
@alena1108 should this commit be from dev or master?

Reply:
@deniseschannon should be updated with master commit once k-d-m changes are merged from dev to master.